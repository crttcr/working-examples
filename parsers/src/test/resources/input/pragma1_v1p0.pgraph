## vertex.auto.id = "GENERATE"
(_:PERSON)