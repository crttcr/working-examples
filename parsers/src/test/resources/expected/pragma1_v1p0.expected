PRAGMA:
	vertex.auto.id=GENERATE
VERTEX: _
	LABEL: PERSON
